examples/harmbench/README.md

```diff
@@ -0,0 +1,50 @@
+# HarmBench
+
+HarmBench is a standardized evaluation framework designed by the Center for AI Safety for automated red teaming of Large Language Models (LLMs). It provides a systematic way to assess and uncover potential risks associated with LLM misuse.
+
+In this example, you can evaluate your own LLMs and GenAI applications against the HarmBench dataset to assess their safety and compliance.
+
+## Setup
+
+1. Set your API keys in your environment. For this particular example, we're going to compare GPT-4o Mini and an intentionally ablated model from [Ollama](https://ollama.ai/) locally. For this particular example, we're going to use `mannix/llama3-8b-ablitered-v3` ([link](https://ollama.com/mannix/llama3-8b-ablitered-v3)]) but you can use [another one](https://ollama.com/search?q=abliterated) of your choice.
```

Comment:
instead of comparing these two it might be more fun to compare an ablated llama to a regular llama.

Reply:
Good idea